packages/eas-cli/src/graphql/queries/PublishQuery.ts

```diff
@@ -0,0 +1,34 @@
+import publishGraphQL from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+
+const PublishQuery = {
+  async getAssetMetadataAsync(
+    assetHashes: string[]
+  ): Promise<{ storageKey: string; status: string; __typename: string }[]> {
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .query<
+          { asset: { metadata: { status: string; storageKey: string; __typename: string }[] } },
+          { assetHashes: string[] }
+        >(
+          publishGraphQL` query {
+          asset {
+            metadata(storageKeys: ${JSON.stringify(assetHashes)}) {
```

Comment:
```suggestion
          gql`query AssetMetadataQuery($assetHashes: [String!]!) {
          asset {
            metadata(storageKeys: $assetHashes) {
```
Please use GraphQL variables instead of string interpolation.